babel.config.js

```diff
@@ -84,7 +93,29 @@ module.exports = function(api) {
           convertESM ? "@babel/transform-modules-commonjs" : null,
         ].filter(Boolean),
       },
-    ],
+      {
+        // The vast majority of our src files are modules, but we use
+        // unambiguous to keep things simple until we get around to renaming
+        // the modules to be more easily distinguished from CommonJS
+        test: [
+          "packages/*/src",
+          "packages/*/test",
+          "codemods/*/src",
+          "codemods/*/test",
+        ],
+        sourceType: "unambiguous",
+      },
+      {
+        // The runtime transform shouldn't process its own runtime or core-js.
+        exclude: [
+          "packages/babel-runtime",
+          /[\\/]node_modules[\\/](?:@babel\/runtime|babel-runtime|core-js)[\\/]/,
```

Comment:
Aren't paths normalized to always use `/` instead of `\\`? If not `[\\/]` should also be used in `@babel\/runtime`.

Reply:
Good question, maybe I just shouldn't worry about it for now then. And you're right that I missed the slash in the middle anyway.

Reply:
Having looked into this now, the input filename is normalized before validation, so on Windows it would be `\\`. Node does not normalize paths to always be `/`.